### ISSUE TITLE
Add. 14,728, foll. 1–75 | https://bl.vuexistapps.us/manuscript/624 (Part1)

### ISSUE THREAD

**Issue by Erfan-Syriaca** [srophe/britishLibrary-data#787]
### Problems with Manuscript Overview

- No date has been mentioned in the MS overview. 

### Categories of Manuscript Overview Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [X] Missing content

### Problems with Physical Description

- The sentence in Hand 1 suggests that the manuscript was written by two scribes. However, only one hand has been mentioned, labeled "sole."
- Content pending
- Content pending
- Accompanying Material is empty. 

### Categories of Physical Description Problems

- [ ] Whitespace
- [ ] Typo(s)
- [X] Incorrect content
- [ ] Missing content

**Comment by mikey-tang16** [comment]
Resolved: empty additions deleted (all entries under Add. 14,728)

Another hand needs to be added, but I'm not sure how, major/minor? Same description as hand 1?
@wlpotter

**Comment by wlpotter** [comment]
Yeah that's tricky...Let's add a second hand note, calling the first major and the second minor (since we need to specify one as major...). And yes, we can repeat the description of hand 1 in hand 2

**Comment by mikey-tang16** [comment]
hand 2 (minor) added
